src/go/k8s/tests/e2e/update-image-tls-client-auth-external-ca/00-assert.yaml

```diff
@@ -1,127 +1,4 @@
-apiVersion: apps/v1
-kind: StatefulSet
+apiVersion: v1
+kind: Namespace
 metadata:
```

Comment:
Namespaces are created by kuttl so tests can be run in parallel. Is there some reason we're not using the kuttl standard here?

Reply:
There is a variable declared in the kuttle environment to check what namespace is being used I think its just $NAMESPACE, can you check to see if that would help?

Reply:
The step creating the namespace has been removed after the rebase.